demo/large_image_demo.py

```diff
@@ -0,0 +1,282 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Please refer to the mmyolo documentation and add relevant instructions to the Chinese and English documentation in `docs/zh_cn/user_guides/inference.md` in mmdet.

Reply:
Please make sure that you have tested all the functionalities and include the test results in the PR message. Thank you.

Reply:
> Please make sure that you have tested all the functionalities and include the test results in the PR message. Thank you.

I have tested all the arguments in large_image_demo, and some necessary results are shown in the PR message. Thank you.

Reply:
> Please refer to the mmyolo documentation and add relevant instructions to the Chinese and English documentation in `docs/zh_cn/user_guides/inference.md` in mmdet.

I have added Chinese and English version in docs.